index.html

```diff
@@ -15,7 +15,135 @@
     <link rel="stylesheet" href="css/style.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
   </head>
-  <body>
+
+  <body id="all">
+    
+
+   <div class="head">
+    <nav class="navbar">
+    <ul class="nav">
+      <li>
+      <a href="#all">Home</a>
+    </li>
+    <li>
+      <a href="#articles">Articles</a>
+    </li>
+    <li>
+      <a href="#po">Popular-posts</a>
+    </li>
+    </ul>
+    </nav>
+
+
+
+
+    <div class="title">
+    <h1 class="elirose">EllieRose Acupuncture Blog</h1>
+    <p class="eli">Welcome to my Blog</p>
+    </div>
+   </div>
+ <main class="flex">
+  <div class="main" id="articles"> 
+    <div class="part1">
+      <div class="acu">
```

Comment:
You can add more than 1 class to any div, so that you do not have so many nested divs i.e `<div class="part1 acu" >` or `<article class="part1 acu" >`